modules/test/playwright/pages/object-web/object-action/EditObjectActionPage.ts

```diff
@@ -51,7 +51,7 @@ export class EditObjectActionPage {
 		this.userPreferredLanguage = page
 			.frameLocator('iframe')
 			.getByText(
-				`Send email notifications in a guest user's preferred language.`
```

Comment:
Hey Larissa thanks so much for noticing this! I believe Brian added the backslash so he could continue using single quotes.

However, I noticed that the formatter removes this backslash and replaces the single quotes with double quotes.

Because he was so careful when deciding which type of quotation mark he would like to use. I believe that the use of hash ( ` ) will not be within the usual range.

Since the formatter adds double quotation marks, I believe we can go this route and remove the hash mark. But maybe I'm exaggerating, I'd like to hear @linolaoj 's opinion on this.

Reply:
Hey @igor-franca! I followed the approach We talked about on our meeting, can you take another look and see if it's alright for you? thanks!

Reply:
Hi @larissacribeiro, I believe it is good to explicit the commit message that it is an **_auto_** source format change. Or say what part of the change was automatically done by the SF.

Reply:
Sure, @linolaoj! Can you take a look now to see if it's clear enough? thanks!